File: single-language-analysis/dataset-files/java/TestSmell017.java
// Original URL: https://github.com/emmai89/Assignment/blob/2ec0af36a6cde4db003a09dbf18ba1ffcf1b1ffa/UnitTest.java#L19-L37

public class TestSmell017 {
   @Test
   public void testLinkedList1()
   {
      int ii;
      for(ii = 0; ii < 20; ii++)
      {
         list.insertLast(ii);
      }
      assertEquals(list.getCount(), 20);

      ii = 0;
      while(!list.isEmpty())
      {
         int test = list.removeFirst();
         assertEquals(test, ii);
         ii++;
      }
      assertEquals(list.getCount(), 0);
   }
}

File: single-language-analysis/dataset-files/python/test_smell_017.py
import pytest

class TestUnitTest:
    def test_smell_017(self):
        ii = 0
        while ii < 20:
            list.insert_last(ii)
            ii += 1
        assert list.get_count() == 20

        ii = 0
        while not list.is_empty():
            test = list.remove_first()
            assert test == ii
            ii += 1
        assert list.get_count() == 0